packages/api/src/formatting.spec.js

```diff
@@ -2,17 +2,17 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import storage from '@polkadot/storage';
+import storage from '@polkadot/storage/testing';
 
 import Api from './index';
 
 const ADDR_ONE = '5EhmTa7fL6SdjgKXo9g6hetR6nHnRAmrtisoGFWEESjzECtY';
 const ADDR_TWO = '5FjbQAHSptRU4Q7Py6zDsRnE7UkpZikJVg5nm1tmu3WuZXKd';
 
-const ENC_ONE = '0x11cf1094db4db43356b7787c3e59c39f';
+const ENC_ONE = '0xe541e84609054b9504b8316ea839b366';
```

Comment:
Obviously still need to double-check these against a running node with e2e tests, i.e. check the balance of Alice with a `--dev` chain. If something horribly large, it works, otherwise, well, we are missing a trick.